static-router-config

```diff
@@ -20,11 +20,12 @@ then
   exit 1
 fi
 
-AWS_ADDRESSES_TO_USE=$(grep --color -A2 'Address' $1 | sed 's/.*:\ \(.*\)$/\1/' | grep '^[0-9].*$' | sed 's/^\(.*\)\/.*$/\1/')
-read -a AWS_ADDRESSES_TO_USE <<< $AWS_ADDRESSES_TO_USE
+AWS_ADDRESSES_TO_USEt=$(grep --color -A2 'Address' $1 | sed 's/.*:\ \(.*\)$/\1/' | grep '^[0-9].*$' | sed 's/^\(.*\)\/.*$/\1/')
+readarray -t AWS_ADDRESSES_TO_USE <<< $AWS_ADDRESSES_TO_USEt
```

Comment:
I think you have a type here `$AWS_ADDRESSES_TO_USEt`

What is the advantage of using read vs readarray by the way?